bin/cml/repository/configure.js

```diff
@@ -0,0 +1,33 @@
+const kebabcaseKeys = require('kebabcase-keys');
+
+const { GIT_USER_NAME, GIT_USER_EMAIL } = require('../../../src/cml');
+
+exports.command = 'configure';
+exports.description = 'Configure the cloned repository';
+
+exports.handler = async (opts) => {
+  const { cml, telemetryEvent: event } = opts;
+  await cml.ci(opts);
+  await cml.telemetrySend({ event });
```

Comment:
this does not exists anymore

Reply:
It **does** exist on the main branch; was it a slip on #1131?

https://github.com/iterative/cml/blob/d6c9a015007ab5ccbcb1fb8fe93d1e32ad36cb81/bin/cml/ci.js#L11

Reply:
Resolved with https://github.com/iterative/cml/pull/1073/commits/d8833fe55fd0a617a483685f0e3cf2ddd517d22d

Reply:
Its not needed anymore due to the last refactor and in any case you should not need to invoke it since its going to be invoked by `bin/cml.js`

Reply:
this is telling us that we do not have enough integration tests